platform/util/platform-util-unavailableurls/src/main/java/org/codice/ddf/platform/util/http/UnavailableUrls.java

```diff
@@ -116,17 +120,33 @@ public boolean contains(String url) {
         }
 
         /**
-         * Computes the next timeout, doesn't exceed MAX_TIMEOUT_SECONDS
+         * Computes the next timeout, doesn't exceed maximum internval
          */
         private void backoff() {
-            timeout = Math.min(timeout * INITIAL_TIMEOUT_SECONDS, MAX_TIMEOUT_SECONDS);
+            // Because the interval doubles, we check against getMaxRetryInterval() / 2.
+            // This prevents a timeout whose period is larger than MAX.
+            if (timeout >= getMaxRetryInterval() / 2) {
+                timeout = getMaxRetryInterval();
+            } else {
+                timeout += timeout;
+            }
         }
 
         private void schedule() {
             getScheduler().schedule(this, timeout, TimeUnit.SECONDS);
             backoff(); // the only reason to schedule is because of a failed ping
         }
 
+        private int getMaxRetryInterval() {
+            String property = System.getProperty(MAX_TIMEOUT_SECONDS_PROPERTY);
```

Comment:
Consider providing a default value with `System.getProperty(MAX_TIMEOUT_SECONDS_PROPERTY, <default-time-out>)` in case the system property isn't set.